.github/workflows/actions.yml

```diff
@@ -0,0 +1,63 @@
+# This workflow will build a Java project with Gradle
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-gradle
+
+name: JUnit Pioneer Flow
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: '*'
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        # test against latest update of each major Java version, as well as specific updates of LTS versions:
+        java: [ 8, 9, 10, 11, 12, 13]
```

Comment:
I'd only keep: `8`, `11`, and introduce `14` as of today.

Reply:
```suggestion
        java: [ 8, 11, 14]
```